src/ui/gql/generated/types.ts

```diff
@@ -309,6 +310,8 @@ export type BuildFlashFirmwareInput = {
 export enum BuildJobType {
   Build = 'Build',
   BuildAndFlash = 'BuildAndFlash',
+  ForceFlash = 'ForceFlash',
+  CheckTarget = 'CheckTarget',
```

Comment:
No, CheckTarget is not currently being used.  I attempted to use it to check the target before performing the actual build and flash, but unfortunately it is not very useful for that purpose because it causes the device to reboot and be disconnected from WIFI, which messes up the flashing process.  I left the job type in the code in case we want to use it for something in the future, but it can be removed.